traduzido/VSCRIPT.txt

```diff
@@ -5212,270 +5366,433 @@ BOLINHOS! ...S-Se você INSISTE. Vamos dizer que ACEITO...!
 
 #v1276Y-Your praise honors me nyeh...! O-Owowowow... 
+S-Seus elogios me deixam honrada, nyeh...! A-Aiaiaii...
 
 #v1277And that ends my spec demonstration nyeh! W-Was that satisfactory nyeh? 
+E com isso acaba minha demonstração de habilidades, nyeh! F-Foi satisfatório, nyeh?
 
 #v0107*cackle*cackle* I like it! I give you permission to serve me! 
+*gargalhadas* Gostei! Te concederei permissão para me servir!
 
-#v0108Oh, that won't do. I almost broke a most precious weapon!! 
+#v0108Oh, that won't do. I almost broke a most precious weapon!!
+Oh, nada bom. Quase quebrei uma arma preciosíssima!! 
 
-#v1278N-Nihi... A fatal error has occurred nyeh. Please contact your service center...nyeh... 
+#v1278N-Nihi... A fatal error has occurred nyeh. Please contact your service center...nyeh...
+N-Nihi... Um erro fatal ocorreu, nyeh. Por favor contate o departamento de atendimento...nyeh...
+—ASSUMINDO: BEATRICE VS CHIESTER410 FIM— 
 
+—ASSUMINDO: BEATRICE VS GEORGE INICIO—
 #v2478Your dance was quite amusing! Polish it up more and come entertain me again! 
+Sua dança foi bem divertida! Melhore ela mais e venha me entreter novamente!
 
 #v2667...So you call my kicks dancing... Have I not been practicing enough...? 
+Então você chama meus chutes de dança... será que não pratiquei o suficiente...?
 
 #v2479Hey now, is that the best a demon king can do? You've really let me down, you know thaaaaat? 
+Ei, isso é o melhor que um rei demônio pode fazer? Você está me desapontando, sabiaaaaa?
 
 #v2668...This is nothing more than a tactical move before my true victory. Someday, you'll see... 
+...Isso não é nada mais que um movimento tático para minha verdadeira vitória. Um dia, você verá...
 
-#v2669Dang... Even if it's against a witch, I can't help but go easy on a woman... 
+#v2669Dang... Even if it's against a witch, I can't help but go easy on a woman...
+Droga... mesmo que seja contra uma bruxa, não consigo evitar pegar leve com uma mulher... 
 
 #v2480You really are a gentleman, aren't you? Groooss, wahyahyahya! 
+Você realmente é um cavalheiro, não é? Nojentooo, wahyahyahya!
+—ASSUMINDO: BEATRICE VS GEORGE FIM—
 
+—ASSUMINDO: BEATRICE VS JESSICA FIM—
 #v2475The fact that you thought to inherit the Headship with power like that makes my sides ache! 
+O fato de que você pensou em herdar a Liderança com um poder como este me faz morrer de rir!
 
 #v2862Shut up! I've got no interest in the Headship! I'll make you pay for this! 
+Cale-se! Não tenho interesse na Liderança! Vou te fazer pagar por isso!
 
 #v2863So, this is the power of Rokkenjima's ruler, the Golden Witch Beatrice... 
+Então, esse é o poder da soberana de Rokkenjima, a Bruxa Dourada Beatrice...
 
 #v2476Hyahyahya! Come now, praise my name! 
+Hyahyahya! Vai, louve meu nome!
 
 #v2477Come on, if the kid's only this strong, just how strong can her parents be? 
+Vamos, se a filha só tem essa força, quão fortes devem ser os pais?
```

Comment:
creio que está dizendo que a filha é forte, então "se a filha tem toda essa força"